### DISCUSSION TITLE
Kierunek rozwoju zawodowego Azure lub M365

### DISCUSSION THREAD

**Discussion by HexorS44** [AzureCommunityPL/group#447] [Ogólna]
Cześć, mam pytanie odnośnie pracy w Polsce z Azure lub M365. Jestem inżynierem informatykiem, pracuję w zawodzie 10 lat. Jestem typowym adminem IT, czyli zajmuje się wszystkim po trochu. Tym samym niczym konkretnym. Od zarządzania serwerami win, linux + wirtualizacja do typowo suportowych tematów. Ostatnio zmigrowałałem pocztę firmową do M365. Najwięcej siedzę w technologiach M$. Mam już swoje lata 82 rocznik i nie chcę się już brać za programowanie (wystarczy mi to ze studiów). Myślę o specjalizacji w kierunku technologii chmurowych Azure i pytanie brzmi: jak u nas w kraju wygląda praca z Azure, przeglądałem oferty na popularnych portalach, wydaje mi się, że nie ma tego za dużo. Zrobiłbym AZ900 i AZ104 na początek. Ciężko znaleźć robotę? Byłbym wdzięczny jak by ktoś mi naświetlił temat w miarę czasu i możliwości. Mam nadzieję, że to odpowiednie miejsce na takie pytanie. Jeśli nie to usunę temat.

**Comment by mgrabarz** [comment]
Zaczynając w nowym obszarze stajesz się w pewnym sensie juniorem. Oczywiście dotychczasowe doświadczenia spoza Azure mocno punktują i szybko ogarniesz nowe tematy. By tego uniknąć uniknąć warto zatrudnić się w dużej organizacji i przykleić do tematów chmurowych, by zrobić pierwszy rok czy dwa doświadczenia.

Nie wiem jak z pracą dla takich "juniorów", ale dla ludzi mających pojęcie w temacie pracy jest mnóstwo i długo jej nie braknie. Stawki też pozwalają przeżyć od pierwszego do pierwszego :)

**Comment by kamil-mrzyglod** [comment]
Czy nie ma tego dużo to bym polemizował. Ostatnio we współpracy z pewną linią lotniczą robiłem rozbudowany kurs na cloud opsa, niedługo startuje kolejny tylko bardziej pod kątem baz danych. Efekt jest taki, że po kursie zatrudnili kilkanaście osób - głównie juniorów. Sam kurs był zrobiony po to, aby w jak nakrótszym czasie zatrudnić jak najwięcej osób do dalszego przeszkolenia już na zasadzie "juniorów z mentorami". Ich głównym problemem był brak chętnych w takiej ilości a nie mało popularna technologia.

Generalnie tak jak Marek napisał - jeśli nie zajmujesz się niczym konkretnym, o dedykowaną ofertę jest trudno. Łatwiej jest szukać pod kątem tematów, które Ciebie zainteresują (czyli pewnie szeroko pojętę "ops") i dopytywać na etapie ofert czy jest tam szansa posiedzieć przy Azure.

**Comment by piotrgwiazda** [comment]
Mamy u nas sporo roboty dla inżynierów chmurowych w rozumieniu stosowania Terraform do budowania infrastruktur Azure, z dobrym rozumieniem bezpieczeństwa i ochrony danych w chmurze. Natomiast nie robimy tego nigdzie w modelu "point and click", tylko Infrastructure as Code.
Na odcinku ochrony i zarządzania tożsamością (Azure AD, conditional access, just in time itp.) oraz pakietów bezpieczeństwa (Azure Defender for Cloud) tematów jest mnóstwo.
(Disclaimer: Zagraniczni klienci, branża finansowa)

**Comment by HexorS44** [comment]
Dziękuję za wszystkie odpowiedzi. Są bardzo pomocne.
